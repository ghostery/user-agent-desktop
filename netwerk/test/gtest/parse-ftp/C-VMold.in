This is a synthetic LISTing based on comments in mirror.pl and a 
screenshot I saw somewhere. Either way, its 'synthetic' because I
couldn't find an FTP server that LISTed like this. Note that (as 
with other VM/CMS LISTings) filesize cannot be determined from 
the listing and (AFAIK) files/dirs not on the 'A' minidisk (see 
'Fm' field) are not RETRievable/CHDIR'able without magic.

Filename FileType  Fm Format Lrecl  Records Blocks Date      Time
LASTING  GLOBALV   A1 V      41     21     1       9/16/91   15:10:32
J43401   NETLOG    A0 V      77     1      1       9/12/91   12:36:04
PROFILE  EXEC      A1 V      17     3      1       9/12/91   12:39:07
DIRUNIX  SCRIPT    A1 V      77     1216   17      1/04/93   20:30:47
MAIL     PROFILE   A2 F      80     1      1       10/14/92  16:12:27
ABADY2K  DATE      A0 V      1      1      1       1/03/100  10:11:12
BBADY2K  DATE      A0 V      1      1      1       11/03/100 10:11:12
AUTHORS            A1 DIR    -      -      -       9/20/99   10:31:11
