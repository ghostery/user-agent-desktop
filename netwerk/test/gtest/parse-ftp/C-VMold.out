<!-- This Source Code Form is subject to the terms of the Mozilla Public
   - License, v. 2.0. If a copy of the MPL was not distributed with this
   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->

09-16-1991  15:10:32                      LASTING.GLOBALV
09-12-1991  12:36:04                      J43401.NETLOG
09-12-1991  12:39:07                      PROFILE.EXEC
01-04-1993  20:30:47                      DIRUNIX.SCRIPT
10-14-1992  16:12:27                      MAIL.PROFILE
01-03-2000  10:11:12                      ABADY2K.DATE
11-03-2000  10:11:12                      BBADY2K.DATE
09-20-1999  10:31:11       <DIR>          AUTHORS
